.test-infra/jenkins/job_bookkeeper_precommit_java8.groovy

```diff
@@ -22,6 +22,11 @@ import common_job_properties
 mavenJob('bookkeeper_precommit_pullrequest_java8') {
   description('precommit verification for pull requests of <a href="http://bookkeeper.apache.org">Apache BookKeeper</a> in Java 8.')
 
+  // Temporary information gathering to see if full disks are causing the builds to flake
+  preBuildSteps {
+    shell("df -h")
```

Comment:
maybe we should add also a "pwd" ? in order to know which is the disk on which the build is running ?

Reply:
Can do. From the test there only is /. But will add

Reply:
Added a ps -ef also, in case something else is also running.